snap/squashfs/squashfs_test.go

```diff
@@ -962,3 +975,27 @@ func (s *SquashfsTestSuite) TestBuildWithCompressionUnhappy(c *C) {
 	})
 	c.Assert(err, ErrorMatches, "(?m)^mksquashfs call failed: ")
 }
+
+func (s *SquashfsTestSuite) TestBuildBelowMinimumSize(c *C) {
+	// this snap is empty. without truncating it to be larger, it should be smaller than
+	// the minimum snap size
+	sn := squashfs.New(filepath.Join(c.MkDir(), "truncate_me.snap"))
+	sn.Build(c.MkDir(), nil)
+
+	size, err := sn.Size()
+	c.Assert(err, IsNil)
+
+	c.Assert(int(size), testutil.IntEqual, int(squashfs.MinimumSnapSize))
```

Comment:
If we use `Equals` here, we don't need to convert the sizes to int
```suggestion
	c.Assert(size, Equals, squashfs.MinimumSnapSize)
```